<launch>
    <!-- 摄像头驱动 -->
    <node pkg="usb_cam" type="usb_cam_node" name="usb_cam">
        <param name="video_device" value="/dev/video0"/>
        <param name="image_width" value="640"/>
        <param name="image_height" value="480"/>
    </node>

    <!-- 节点启动 -->
    <node pkg="book_navigator" type="barcode_reader.py" name="barcode_reader"/>
    <node pkg="book_navigator" type="database_handler.py" name="database_handler"/>
    <node pkg="book_navigator" type="navigation_controller.py" name="navigation_controller"/>
    <node pkg="book_navigator" type="serial_communicator.py" name="serial_communicator"/>
    
</launch>